src/opencolorio-1-fixes.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 13e2d64..47b5297 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,6 +9,12 @@ if(NOT DEFINED CMAKE_FIRST_RUN)
+     SET(CMAKE_FIRST_RUN 1 CACHE INTERNAL "")
+ endif()
+ 
++if (MSVC)
++    set(PATCH myptch)
++else()
++    set(PATCH patch)
```

Comment:
MXE has Makefile variable `$(PATCH)`. "patch" is not the only possible value of it. You should pass the value from build rules to CMake as a CMake variable and use it here.